### ISSUE TITLE
Retraining image to primitive model

### ISSUE THREAD

**Issue by askaradeniz** [PrincetonLIPS/vitruvion#3]
Hi, thanks for your interesting and impressive work!

I'm curious about what can be the possible reasons that I'm getting a low-score for re-training image-to-primitive model. All the hyperparameters were the same with the original model and I adapted your slurm script to run the model on a similar setup (4 v100 gpus). Final accuracy for the image to primitive model was around 0.47.

I used the following  slurm script to retrain the model.
```shell
#!/bin/bash -l

#SBATCH -p gpu
#SBATCH -N 1
#SBATCH --gpus=4
#SBATCH --constraint=volta32
#SBATCH --job-name train_image_to_primitive
#SBATCH --time=24:00:00
source activate vitruvion

python -m img2cad.train_image_to_primitive +cluster=akaradeniz +compute=4xv100 batch_size=2048
```

These are the graphs from the training logs:

![Screenshot from 2022-09-12 10-01-25](https://user-images.githubusercontent.com/50996598/189602806-847ae339-8e4b-4101-9c17-bd23975d1cdf.png)

Thank you in advance!

**Comment by tarun550** [comment]
Can you please share the torch, torch-lightning version used?

**Comment by askaradeniz** [comment]
Hi, I don't remember the torch and torch-lightning version used for the issue, but it was resolved after training with the provided singularity.

**Comment by tarun550** [comment]
I understand, but now the singularity image is not available for download- i am trying to reproduce the results but the version differences are causing issues

can you please help?
